core/templates/dev/head/player/flag_exploration_popup.html

```diff
@@ -0,0 +1,42 @@
+<script type="text/ng-template" id="modals/flagExploration">
+  <div class="modal-header">
+    <h3>
+      <span>Report Exploration</span>
+    </h3>
+  </div>
+  <div class="modal-body oppia-long-text">
+    <form>
+      <div class="radio">
+        <label>
+          <input type="radio" value="AD" ng-model="flag" ng-change="triggerOtherFlagDropdown(flag)">
+          It is an Ad
+        </label>
+      </div>
+      <div class="radio">
+        <label>
+          <input type="radio" value="inappropriate" ng-model="flag" ng-change="triggerOtherFlagDropdown(flag)">
+          Inappropriate Content
+        </label>
+      </div>
+      <div class="radio">
+        <label>
+          <input type="radio" value="other" ng-model="flag" ng-change="triggerOtherFlagDropdown(flag)">
+          Other
+        </label>
+        <p ng-show="otherFlagDropdown">
+        Please enter a brief description of what you want to report about the exploration:
+          <textarea rows="3" cols="50" ng-model="flagMessage">
+          </textarea>
+        </p>
+      </div>
+    </form>
+  </div>
+  <div class="modal-footer">
+    <button class="btn btn-default" ng-click="cancel()">
+    Cancel
+    </button>
+    <button class="btn btn-success" ng-click="submitReport()">
```

Comment:
This button should be disabled if nothing is selected, or if "other" is selected and no reason is supplied.